### DISCUSSION TITLE
Dark mode on Gnome/GTK

### DISCUSSION THREAD

**Discussion by ramonsmits** [audacious-media-player/audacious#1527] [Q&A] [answered]
I'm using Audacious on Fedora 41 and installed it via the RPM package manager. 

I'm a "dark mode" user and noticed Audacious not using it. I can't locate a setting but I thought because it uses GTK it should automatically use this.  

Does this require something on my end or does this need a change in Audacious to support dark mode?

**Comment by radioactiveman** [comment]
You may need to set the environment variable `GTK_THEME` to a dark theme, for example with `export GTK_THEME=Adwaita:dark`.

Another option is to use [gnome-tweaks](https://wiki.gnome.org/Apps/Tweaks) where the theme for GTK 3 applications can be set under `Appearance -> Legacy applications`.

![gnome-tweaks](https://github.com/user-attachments/assets/9eb92df5-8b94-40ca-ae4f-14ca3f3ef6ee)

**Reply by ramonsmits** [reply, marked as answer]
Interesting, I don't have that Yaru-Dark in Tweaks. 

![image](https://github.com/user-attachments/assets/2398c15e-c2be-4177-9a63-b2c9c20a17e6)

So I learned that Audacious is based on GTK3 and Fedora 41 uses GTK 4:

I have 3 options:

- Adwaita
- HighContrast
- HighContrastInverse

The last pretty much seems like dark mode. Changing these immediate updates Audacious. Keeping it on `HighContrastInverse` shows me this:

![image](https://github.com/user-attachments/assets/97b842f3-928e-47a8-a61f-d684fca374d5)



> You may need to set the environment variable `GTK_THEME` to a dark theme, for example with `export GTK_THEME=Adwaita:dark`.

After googling on `GTK_THEME` I found that GTK3 settings are stored in `~/.config/gtk-3.0/settings.ini` and I updated it to:

```ini
[Settings]
gtk-application-prefer-dark-theme=1
```

That enabled Dark mode for me!

**Reply by radioactiveman** [reply]
My screenshot is from Ubuntu, Yaru is its default theme. Glad you found a solution. :+1:

https://www.baeldung.com/linux/gtk-applications-dark-theme gives some more information about it.
